src/ess/livedata/handlers/to_nxlog.py

```diff
@@ -42,9 +45,13 @@ def _at_capacity(self) -> bool:
     def _ensure_capacity(self, data) -> None:
```

Comment:
```suggestion
    def _ensure_capacity(self, data: sc.Variable) -> None:
```
Would have helped me understand the change.